scripts/release/update-slugs-table.js

```diff
@@ -7,15 +7,16 @@
 const fs = require("fs");
 const path = require("path");
 
-const dataFile = path.resolve(__dirname, "..", "_data", "simple-icons.json");
-const slugsFile = path.resolve(__dirname, "..", "slugs.md");
+const rootDir = path.resolve(__dirname, "..", "..");
+const dataFile = path.resolve(rootDir, "_data", "simple-icons.json");
+const slugsFile = path.resolve(rootDir, "slugs.md");
 
 const data = require(dataFile);
-const { getIconSlug } = require("./utils.js");
+const { getIconSlug } = require("../utils.js");
 
 let content = `<!--
 This file is automatically generated. If you want to change something, please
-update the script at '${__filename.replace(__dirname, "scripts")}'.
+update the script at '${__filename.replace(__dirname, "scripts/release")}'.
```

Comment:
This could be simplified:

```suggestion
update the script at '${path.relative(rootDir, __filename)}'.
```

Reply:
Nice, I like it :+1: